### PR TITLE
feat: Prettier boilermaker update (remember to npm install after merge)

### PR DESCRIPTION
**Pull request by Delune**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  presets: ['@babel/react', '@babel/env', '@babel/stage-2']
+  presets: ['@babel/react', '@babel/env', ["@babel/preset-stage-2", { "decoratorsLegacy": true }]]
```
Comment (mdfergus) [inline .babelrc:2]: Pretty

**Review by mdfergus — DISMISSED** [review]

**Review by antonholmes — DISMISSED** [review]
Goodbye Semicolons!;;;

**Review by antonholmes — APPROVED** [review]
Looks good

**Review by mdfergus — APPROVED** [review]
So pretty

**Pull request merged**